cabal-install/src/Distribution/Deprecated/ViewAsFieldDescr.hs

```diff
@@ -5,7 +5,7 @@ module Distribution.Deprecated.ViewAsFieldDescr
 import Distribution.Client.Compat.Prelude hiding (get)
 import Prelude ()
 
-import qualified Data.List.NonEmpty as NE
+import Data.List (minimumBy)
```

Comment:
```suggestion
import Data.Foldable (minimumBy)
```
...otherwise it feels weird to import `minimumBy` from `Data.List`, but apply it to `NonEmpty`.

Reply:
Thanks for the suggestion @Bodigrim.